grip/browser.py

```diff
@@ -0,0 +1,14 @@
+import socket
+import webbrowser
+
+
+def start_browser(host, port):
+    sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    #waiting for server to start
+    while sock.connect_ex((host, port)) not in [0, 22]:
```

Comment:
It would be more explicit to use `errno.EINVAL` instead of hand written number, besides, why invalid argument error considered OK to open browser here?